app/scripts/contentscript.js

```diff
@@ -928,10 +946,40 @@ function prepDiff(spdxid, time, html, details) {
   if (details.licenseComments) {
     hoverInfo += "&#10;comments: " + _.escape(details.licenseComments);
   }
-  var title = `<a href="https://spdx.org/licenses/${spdxid}.html" target="_blank" title="${hoverInfo}">${details.name} (${spdxid})</a>`;
+
+  let confidenceIndicator = '';
+  if (percentage !== undefined) {
+      let confidenceText = '';
+      let confidenceClass = '';
+      let icon = '';
+
+      if (percentage >= 90) {
```

Comment:
Instead of hard coding, can we create an array for thresholds? Maybe put in const.js?